pkg/kapp/cmd/app/deploy_flags.go

```diff
@@ -45,6 +47,9 @@ func (s *DeployFlags) Set(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&s.OverrideOwnershipOfExistingResources, "dangerous-override-ownership-of-existing-resources",
 		false, "Steal existing resources from another app")
 
+	cmd.Flags().BoolVar(&s.DisableDefaultLabelScopingRules, "disable-default-label-scoping-rules",
```

Comment:
not sure if we have other similar flags for tweaking config... may be default-label-scoping-rules=true (as a default)

Reply:
> not sure if we have other similar flags for tweaking config

Yeah, I don't think we have any other similar flag.

> may be default-label-scoping-rules=true (as a default)

Definitely make more sense. I will make the change.